vagrant/config/docker/config.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
I think it's necessary.
vagrant/docker/ova they may need have different config **for test**.

For this docker/config.json, all ```172.31.128.1``` is changed to ```172.31.128.250```, which is the ture em1 IP of our jenkins slaves.